traefik/traefik.yaml.j2

```
@@ -33,6 +33,14 @@ certificatesResolvers:
   {%- elif letsencrypt.challenge_type == 'dnsChallenge' %}
       dnsChallenge:
         provider: {{ letsencrypt.provider }}
+        delayBeforeCheck: {{ letsencrypt.delayBeforeCheck }}
+      {% if letsencrypt.resolvers %}
+        resolvers:
+        {% for resolver in letsencrypt.resolvers -%}
+          - "{{ resolver }}"
+
+        {% endfor %}
```

Please remove the empty line and you can add an `-` to the `endfor` within the yaml loop to keep it all neatly tight together.

```suggestion
        {%- endfor %}
```